thunder/core/jit_ext.py

```diff
@@ -908,12 +908,16 @@ def is_from_torch(fn):
             # Torch functions have __name__ defined
             fn_name = f"{fn.__module__}.{fn.__name__}"
 
-            # For now, only torch-like opaque functions are sharp edges
-            return _general_jit_sharp_edge(
-                f"Trying to call function {fn_name}, but it's unsupported. Please file an issue requesting support.",
-                None,
+            # Probably merge with sharp edges
+            get_calling_opaque_torch_msg = (
+                lambda: f"Trying to call function {fn_name}, but it is not yet supported. "
```

Comment:
Why does this need to be a lambda?

Reply:
Future proofing for sharp edges

Reply:
Sorry, this is the terminal path, will fix that.